src/adapter/src/coord.rs

```diff
@@ -1009,17 +1009,20 @@ pub async fn serve<S: Append + 'static>(
             for (timeline, initial_timestamp) in initial_timestamps {
                 let oracle = if timeline == Timeline::EpochMilliseconds {
                     let now = now.clone();
+                    let initial_timestamp = std::cmp::max(initial_timestamp, (now)().into());
                     handle.block_on(timeline::DurableTimestampOracle::new(
                         initial_timestamp,
                         move || (now)().into(),
                         *timeline::TIMESTAMP_PERSIST_INTERVAL,
+                        *timeline::TIMESTAMP_MAX_INCREASE,
                         |ts| catalog.persist_timestamp(&timeline, ts),
                     ))
                 } else {
                     handle.block_on(timeline::DurableTimestampOracle::new(
                         initial_timestamp,
                         Timestamp::minimum,
                         *timeline::TIMESTAMP_PERSIST_INTERVAL,
+                        *timeline::TIMESTAMP_MAX_INCREASE,
```

Comment:
Having non-realtime timelines limit their advancement to 10s at a time is wrong. Non-realtime should probably be exempt from this feature alltogether.

Reply:
Good point, I switched this to be an `Option` so it only affects realtime timelines.